yuhow/yuhowNote.tex

```diff
@@ -62,11 +240,261 @@ \section{Something about Python in Solvcon}
   \item PyCharm
   \item iPython
  \end{enumerate}
+And the complete Python code of CESE solver can be found in Section~\ref{subsec:python_code}. Section~\ref{sec:1d_shock_tube} will  demonstrate how we implement CESE solver for solving this one-dimensional shock tube problem in Python.
+
+\subsection{One-dimensional shock tube problem}
+ \label{sec:1d_shock_tube}
+
+The one-dimensional Sod shock tube is employed for the test of our CESE computational fluid codes. Figure~\ref{fig:sod_shock_tube_1D} shows the initial condition of different region in our Sod shock tube. The whole one-dimension space is separated into two regions by a diaphragm in the beginning. 
+The left region is called driven section and containing the gas with the higher density $\rho_l$ and the higher pressure $p_l$. The right region is called working section and with the gas at the lower density $\rho_r$ and the lower pressure $p_r$. Before removing the diaphragm, the gas in both regions is static. 
+The global heat capacity ratio ($\gamma\equiv\frac{C_p}{C_v}$) is 1.4.
```

Comment:
$\gamma = \frac{c_p}{c_v}$ is called specific heat ratio.  "Heat capacity ratio" is a wrong name.  $c_p$ is specific heat at constant pressure while $c_v$ is specific heat at constant volume.  Note it's lower-cased "c", not upper-case "C".  See [1].

[1] J. D. Anderson, Modern Compressible Flow: With Historical Perspective, 3rd ed. Boston: McGraw-Hill, 2003. http://www.amazon.com/Modern-Compressible-Flow-Historical-Perspective/dp/0072424435